rules/windows/execution_scripts_process_started_via_wmi.toml

```diff
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2020/11/27"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of the scripting engine cscript.exe or wscript.exe to execute a process via Windows Management
+Instrumentation. This may be indicative of malicious activity.
```

Comment:
```suggestion
Identifies use of the built-in Windows script interpreters (cscript.exe or wscript.exe) being used to execute a process via Windows Management
Instrumentation. This may be indicative of malicious activity.
```

Reply:
may need to re-lint